configs/components/puppet.json

```diff
@@ -1 +1 @@
-{"url":"https://github.com/openvoxproject/puppet.git","ref":"91c19d89ea9a585026230c12f874e8f7d4e77c57"}
+{"url":"https://github.com/openvoxproject/puppet.git","ref":"7e684ed800c149a735ab1176faa9afddb476d5b1"}
```

Comment:
```suggestion
{"url":"https://github.com/openvoxproject/puppet.git","ref":"58ad649671153adb975d382bcca51d66331f42a7"}
```

Reply:
I thought for the last 7 release the commit that bumped the version was used, not the merge commit (and I think ewoud prefers that?). But on the merge commit is the git tag, and referencing that makes sense as well.